proxy/handler.go

```diff
@@ -88,13 +90,13 @@ func (s *handler) handler(srv interface{}, serverStream grpc.ServerStream) error
 				// this is the happy case where the sender has encountered io.EOF, and won't be sending anymore./
 				// the clientStream>serverStream may continue pumping though.
 				clientStream.CloseSend()
-				break
+				continue
```

Comment:
Does this fix an issue?

Reply:
Probably not in this context. I ran `staticcheck` and fixed [check SA4011](https://staticcheck.io/docs/checks#SA4011) here.